mountpoint-s3-client/examples/client_benchmark.rs

```diff
@@ -115,6 +115,13 @@ struct CliArgs {
         visible_alias = "maximum-throughput-gbps"
     )]
     throughput_target_gbps: f64,
+    #[arg(
+        long,
+        help = "CRT Memory limit in GB",
+        default_value = "0",
+        visible_alias = "memory-limit-gb"
+    )]
+    crt_memory_limit_in_gb: u64,
```

Comment:
nit: wouldn't include the "in"

```suggestion
    #[arg(
        long,
        help = "CRT Memory limit in GB",
        default_value = "0",
        visible_alias = "memory-limit-gb"
    )]
    crt_memory_limit_gb: u64,
```